Packs/FeedTAXII/ReleaseNotes/1_1_4.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### TAXII 2 Feed
+- Fixed an issue in **TAXII2ApiModule** where **extension-definition** objects arrived to the system instead of being skipped.
+- Fixed an issue in **TAXII2ApiModule** where an error message was thrown when not needed for server 2.1 connection.
```

Comment:
i think we can remove that point, im afraid it might be more confusing than actually explain

what do you think ?

Reply:
Yeah might be, I'll remove it :)